core/execution/market_snapshot.go

```diff
@@ -190,6 +195,12 @@ func (m *Market) getState() *types.ExecMarket {
 		sp = m.settlementDataInMarket.Clone()
 	}
 
+	parties := make([]string, 0, len(m.parties))
+	for p := range m.parties {
+		parties = append(parties, p)
+	}
```

Comment:
Just style so be free to ignore, but I've started to like doing these: `parties := maps.Keys(m.parties)`, it ultimately does exactly the same as your code.

Reply:
Thats nice. Changed.